website/static/css/mailing-list-modal.css

```diff
@@ -0,0 +1,17 @@
+.link-url{
+    padding: 3px 0px;
+    background-color: #F5F5F5;
+}
+
+.btn-no-pad{
```

Comment:
This rule should not be necessary; we already have the `copy-button` class. You may have to move that class to `style.css` and make it more specific, e.g. `.copy-button, button.copy-button`, so that it takes precedence over other rules.